_events/discussions/event-013.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Research Software Directories: What, Why and How?"
+authors:
+    - &speaker
+      name: Mark Woodbridge
+      orcid: 0000-0002-0650-9751
+      is_speaker: true
+      affiliation: 1
+    - name: Vanessa Sochat
+      orcid: 0000-0002-4387-3819
+      is_speaker: true
+      affiliation: 3
+    - name: Jurriaan Spaaks
+      orcid: 0000-0002-7064-4069
+      is_speaker: true
+      affiliation: 2
+affiliations:
+    - name: Imperial College London, UK
+      index: 1
+    - name: Netherlands eScience Center, The Netherlands
+      index: 2
+    - name: Stanford University, US
+      index: 3
+author: *speaker
+category: discussions
```

Comment:
@eileen-kuehn, I just realized: is [event-013](https://887-267395254-gh.circle-artifacts.com/0/SORSE/programme/discussions/event-013/index.html) a panel discussion? If so, then the correct team identifier is `panel`, not `discussions`, see https://github.com/SORSE/sorse.github.io/blob/master/_data/committee/programme_teams.yml
```suggestion
category: panels
```

Reply:
Hm...  that is even more complicated. It seems to be a *panel*, but was reviewed as a *discussion*.

Reply:
I got it now. We have deviating options for contributions and tracks (tracks are needed for reviewing). The contribution types are more fine-grained than the tracks. I defined the tracks along the options on the website whereas the contribution types handle everything on its own. See image.

<img width="611" alt="Screenshot 2020-08-20 at 08 14 35" src="https://user-images.githubusercontent.com/8090701/90723553-3be85680-e2bd-11ea-9fbd-bab038c73d0d.png">

So I think we should drop some of the contribution types? @cc-a, what do you think?
Internally I can just several contribution types to the same tracks to fix this concrete issue.

Reply:
Alternatively we can also add a _discussions_ to the website. This was just not part of the original programme teams

Reply:
I must have missed that we have a _Discussion session_ category. Who is part of that review team?

Reply:
Yep they have a combined remit for both panels and discussions - https://docs.google.com/document/d/16OZLKW_SM0VvWpPHPnEbhbNbs6mzCvOdXB3zppsH1n4/edit

Reply:
then maybe it's the best to add a new programme team to the webpage. We'd just need a little text to go on https://sorse.github.io/programme/discussions/ (such as in https://sorse.github.io/programme/panels/)

Reply:
shall I ask on slack? We can revert ae10621 and merge this PR, even if the `discussions` page is still missing

Reply:
So I don't think we want to drop any contribution types. Can the `discussions` page just be a redirect to the `panels` page as this covers both?

Reply:
hmm, why not just take the text from the `panels` page and put it on the `discussions` page. This would acknowledge the two different contribution types and it would be consistent with all the remaining teams.

Reply:
Whichever is easiest I guess...